cli/cmd/wallet/cmd/public-key/flag/generate.go

```diff
@@ -0,0 +1,32 @@
+package flag
+
+import (
+	"github.com/bloxapp/eth-key-manager/cli/util/cliflag"
```

Comment:
please, sort imports:
1. build in packages
2. third party packages 
3. internal packages